### DISCUSSION TITLE
Responsive layout for smartphones or mobile devices

### DISCUSSION THREAD

**Discussion by ajsb85** [uxbox/uxbox#1295] [Ideas]
Dearest.
We want responsive layout for smartphones or mobile devices.
To design with iPad pro.
Regards.

**Comment by elhombretecla** [comment]
Hi @ajsb85 Thanks for your feedback :)

We just create an enhancement to manage this ar Taiga. You can track it progress here

[https://tree.taiga.io/project/penpot/issue/2201](https://tree.taiga.io/project/penpot/issue/2201)
